improve image and vision support (#2935)
pipecat-ai/pipecat: CHANGELOG.md

## [Unreleased]

### Added

- Added `VisionImageRawFrame`. This is an input image frame with an associated
  text. It is usually processed by vision services (e.g. Moondream). The text

>   indicates the vision service what should it describe.

Reviewer: ```suggestion
guides the vision service on how to analyze the image.
```

- Added support for including images or audio to LLM context messages using
  `LLMContext.create_image_message()` or `LLMContext.create_image_url_message()`

>   (not all LLM support URLs) and `LLMContext.create_audio_message()`. For

Reviewer: ```suggestion
  (not all LLMs support URLs) and `LLMContext.create_audio_message()`. For
```
